proposals/handler.go

```diff
@@ -325,19 +330,11 @@ func (h *Handler) handleProposal(ctx context.Context, expHash types.Hash32, peer
 	proposalDuration.WithLabelValues(decodeInit).Observe(float64(time.Since(t0)))
 
 	logger = logger.WithFields(p.ID(), p.Ballot.ID(), p.Layer)
-	t1 := time.Now()
-	if has, err := proposals.Has(h.db, p.ID()); err != nil {
-		logger.With().Error("failed to look up proposal", log.Err(err))
-		return fmt.Errorf("lookup proposal %v: %w", p.ID(), err)
-	} else if has {
+	if h.proposals.IsKnown(p.Layer, p.ID()) {
 		known.Inc()
 		return fmt.Errorf("%w proposal %s", errKnownProposal, p.ID())
 	}
-	proposalDuration.WithLabelValues(dbLookup).Observe(float64(time.Since(t1)))
 
-	logger.With().Info("new proposal",
-		log.String("exp hash", expHash.ShortString()),
-		log.Int("num_txs", len(p.TxIDs)))
```

Comment:
do we log it somewhere else? why to remove?

Reply:
It's logged in `Hare::OnProposal()`

Reply:
one thing is that it is logged with Debug now, i think it is better to keep it with Info in the handler itself

Reply:
Sorry, I missed your comment. I created a PR with it:  #5610